js/foundation.accordion.js

```diff
@@ -165,16 +155,16 @@ class Accordion {
 
   /**
    * Closes the tab defined by `$target`.
-   * @param {jQuery} $target - Accordion tab to close.
+   * @param {jQuery} $target - Accordion tab to close (`.accordion-content`).
    * @fires Accordion#up
    * @function
    */
   up($target) {
     var $aunts = $target.parent().siblings(),
         _this = this;
-    var canClose = this.options.multiExpand ? $aunts.hasClass('is-active') : $target.parent().hasClass('is-active');
+    var canClose = $aunts.hasClass('is-active');
 
-    if(!this.options.allowAllClosed && !canClose) {
+    if(!this.options.allowAllClosed && !$aunts.hasClass('is-active')) {
```

Comment:
I'm a little confused by this change... do we not need to check if the target is open?  And if not, why do we need the 'canClose' variable at all?